src/pages/auth/signup.tsx

```diff
@@ -28,12 +28,12 @@ const schema = z.object({
   password: z
     .string()
     .min(8, { message: "Password should be at least 8 characters long" }),
-  confirmPassword: z.string(),
+  confirmPassword: z.string().min(8, { message: "Password should be at least 8 characters long" }),
 })
-.refine(data => data.confirmPassword === data.password, {
+.refine((data) => data.confirmPassword === data.password, {
   message: "Passwords don't match",
-  path: ['confirm'],
-})
+  path: ['confirmPassword'],
```

Comment:
Don't think that we need this (`path: [...`) at all. Let's just remove this.

Reply:
I'm not sure what this ```path``` thing does, I left it in just in case, but I'll remove it.

Reply:
It just shows where the error originated. It is usable when you are working with zod directly. Thank you!

Reply:
No problem!